### DISCUSSION TITLE
How many assets do I have?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#8391] [General]
Hi, 

Is there a way to see how many assets a website has?

Initially I'd like to simply check my local/staging/production sites all have the same number of assets in `/public/assets/`

I exclude `/public/assets/` from git, however I have recently started using the excellent Laravel Sync by Michael Aerni.

It'd be a useful peace of mind check.

It'd be handy to group assets by type (.jpg, .pdf etc)

**Comment by duncanmcclean** [comment]
I don't know if there's a way of doing this within the Control Panel but you can do it by running a query in `php artisan tinker` (or Tinkerwell if you use that).

Run this for every asset container you need to check:

```php
\Statamic\Facades\Asset::query()->where('container', 'assets')->count();
```

**Reply by stephenmeehanuk** [reply]
That's really handy, thanks!
